File: universal/src/main/java/com/github/fernthedev/exceptions/DebugException.java
package com.github.fernthedev.exceptions;

public class DebugException extends Exception {

    public DebugException() { }

    public DebugException(String message) {
        super(message);
    }

    public DebugException(Throwable throwable) {
        super(throwable);
    }
}

File: csharp/core/exceptions/DebugException.cs
using System;
using System.Runtime.Serialization;

namespace com.github.fernthedev.lightchat.core.exceptions
{
    public class DebugException : SystemException
    {
        public DebugException()
        {
        }

        protected DebugException(SerializationInfo info, StreamingContext context) : base(info, context)
        {
        }

        public DebugException(string? message) : base(message)
        {
        }

        public DebugException(string? message, Exception? innerException) : base(message, innerException)
        {
        }
    }
}